docs/tasks/istio-installation.md

```diff
@@ -7,31 +7,106 @@ layout: docs
 type: markdown
 ---
 {% capture overview %}
-This task shows how to install Istio in a Kubernetes cluster. You'll learn
-how to install Istio and get it initially configured and running.
+This page shows how to install and configure Istio in a Kubernetes cluster.
 {% endcapture %}
 
 {% capture prerequisites %}
-* Do this.
-* Do this too.
+* The following instructions assume you have access to a Kubernetes cluster. To install Kubernetes locally, try [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+* If you are using [Google Container Engine](https://cloud.google.com/container-engine), please make sure you are using static client certificates before fetching cluster credentials:
+```bash
+gcloud config set container/use_client_certificate True
+gcloud container clusters get-credentials cluster-2 --zone us-central1-a --project istio-demo
+```
+* Ensure the curl command is present, or
 {% endcapture %}
 
 {% capture steps %}
-## Doing ...
+## Installing on an existing cluster
+For the Alpha release, Istio must be installed in the same Kubernetes namespace as the applications. Instructions below will deploy Istio in the "default" namespace. They can be modified for deployment in a separate namespace.
 
-1. Do this.
-1. Do this next. Possibly read this [related explanation](...).
+1. a. Download and extract the [istio installation files](https://raw.githubusercontent.com/istio/istio/master/releases/istio-alpha.tar.gz), or\
+b. Clone the github [istio](https://github.com/istio/istio) repository:
+```bash
+git clone https://github.com/istio/istio
+```
+1. Change directory to istio:
+```bash
+cd istio
+```
+2. Install the Istio core components (Istio-Manager, Mixer, and Ingress-Controller):
+```bash
+kubectl apply -f ./kubernetes/istio.yaml
+```
+1. Source the Istio configuration file:
+```bash
+source istio.VERSION
+```
```

Comment:
This sourcing of istio.VERSION complicates install. Since the user will either download Istio files or clone a specific version, this step should not be needed, since the local files will always point to a specific version. Agree?